lib/schema_plus/foreign_keys/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'abstract/schema_creation'
```

Comment:
I'd move this `require_relative` next to all the others in `lib/schema_plus/foreign_keys.rb`